.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "config/plugins/unipept"]
+	path = config/plugins/unipept
+	url = https://github.com/caleb-easterly/galaxy-plugin-unipept-visualization.git
```

Comment:
Thanks for the contribution! Did you mean for this to be here - the project as a submodule? It looks like you tried to add it as a submodule and then just added it directly - is that correct? I'd prefer not adding submodules to Galaxy for now.